core/controllers/get_filters.py

```diff
@@ -32,3 +32,18 @@ def get_funds():
     funds = [{"name": FUND_ID_TO_NAME[fund_id], "id": fund_id} for fund_id in fund_ids]
 
     return funds, 200
+
+
+def get_outcome_categories():
+    """Returns a list of all outcome categories.
+
+    :return: List of outcome categories
+    """
+    outcome_dims = OutcomeDim.query.with_entities(OutcomeDim.outcome_category).distinct().all()
+
+    if not outcome_dims:
```

Comment:
is this definitely how we want it to behave? (it probably is, I'm just checking). As there's a real chance we won't have any Categories stored (ie the column may have nan's in if we don't get given some mappings), in this (hopefully unavoidable) case, should we get an empty list? Or will the error be handled?

Reply:
I don't know if it'd been decided as such but this follows the existing pattern on these endpoints. I guess it doesn't really matter as long as the error is handled. I would imagine we'd want to handle it in the same way for all of these filters. The bit of work for calling this endpoint from the front-end hasn't been done yet so I'll put a note on the goal and related tickets.